rasa/nlu/utils/hugging_face/hf_transformers.py

```diff
@@ -449,7 +602,10 @@ def _get_model_features_for_batch(
         return sentence_embeddings, sequence_final_embeddings
 
     def _get_docs_for_batch(
```

Comment:
Just for my understanding: Did we also update the call of `_get_docs_for_batch` in `train` and `predict`?

Reply:
But then how does the method know it is in inference mode or not? Sorry I am a bit confused.

Reply:
You are absolutely right! I had missed passing that argument in the call to `_get_docs_for_batch` in `process` method. The call in `train` stays stays the same since its False by default. I passed them in the tests correctly so, didn't catch the problem.
Thanks for catching this.

Reply:
Can/Should we add a test for this?